docs/contributing/docs-templates.md

```diff
@@ -202,9 +202,11 @@ A recipe should list requirements and include a few short instructions to comple
 The components of a recipe are:
 
 - The name of the recipe, which should describe a single task
-- A list of requirements and prerequisites
+- A 1-2 sentence description motivating what the recipe is for
+- Prerequisites and requirements
 - Step-by-step directions
-- A link to a working example
+  - Optional embedded examples
```

Comment:
Was this supposed to be indented?

Reply:
Yep, it's intentionally under "Step-by-step directions" since that's where those would go.